crates/starknet-server/src/api/json_rpc/endpoints.rs

```diff
@@ -149,10 +162,14 @@ impl JsonRpcHandler {
         let starknet = self.api.starknet.read().await;
         match starknet.get_class_hash_at(&block_id.into(), &contract_address.0) {
             Ok(class_hash) => Ok(FeltHex(class_hash)),
-            Err(Error::BlockIdHashUnimplementedError | Error::BlockIdNumberUnimplementedError) => {
-                Err(ApiError::BlockNotFound)
+            Err(
+                Error::BlockIdHashUnimplementedError
```

Comment:
Can this still be an error? Can the error definitions be removed?

Reply:
also on the call method on L208 in endpoints.rs

Reply:
[3074c87](https://github.com/0xSpaceShard/starknet-devnet-rs/pull/104/commits/3074c877e437aa6c811442fe2bf8be2b33ef7124)